website/pages/en/support.js

```diff
@@ -14,8 +14,9 @@ The React Native for Windows (RNW) Team strives to provide full support for the
 | Version | Support Phase | Release Date | Active Support Start | Maintenance Support Start | End of Support |
 | -- | -- | -- | -- | -- | -- |
 | [main](https://www.npmjs.com/package/react-native-windows/v/canary) | [Canary](#canary-support) | *N/A* | *N/A* | *N/A* | *N/A* |
-| [0.78](https://www.npmjs.com/package/react-native-windows/v/latest) | [Active](#active-support) | 02/27/2025 | 02/27/2025 | *TBD* | *TBD* |
-| [0.77](https://www.npmjs.com/package/react-native-windows/v/v0.77-stable) | [Maintenance](#maintenance-support) | 01/24/2025 | 01/24/2025 | 03/27/2025 | 05/31/2025 |
+| [0.79](https://www.npmjs.com/package/react-native-windows/v/latest) | [Active](#active-support) | 06/17/2025 | 06/17/2025 | *TBD* | *TBD* |
+| [0.78](https://www.npmjs.com/package/react-native-windows/v/v0.78-stable) | [Maintenance](#maintenance-support) | 02/27/2025 | 02/27/2025 | 07/17/2025 | 09/20/2025 |
```

Comment:
When adding the Active/Maintenance support start dates should be rounded to the last day of the month, and 0.78 is still active.

```suggestion
| [0.78](https://www.npmjs.com/package/react-native-windows/v/v0.78-stable) | [Active](#active-support) | 02/27/2025 | 02/27/2025 | 07/31/2025 | 09/30/2025 |
```